docs/schemas/v1/definitions.json

```diff
@@ -82,6 +83,26 @@
                     ]
                   },
                   "description": "Available roles for this application. These can be assigned to users."
+                },
+                "roleMappings": {
```

Comment:
Intuitively this feels like a provider specific property. e.g. how would this work under Google/GitHub?
I'd sort of have expected something like:

```yml
providers:
  - provider: 'azure-ad'
    roles:
      - source: 
          - my-azure-role
          - my-other-azure-role
        target: my-toolpad-role
```

Also, UI doesn't really have proirity right now. I'm fine to release this with config only

Reply:
You're right, I'll make that change, seems better. The UI is already done so should be easy to keep now.

Reply:
I've changed the schema here: https://github.com/mui/mui-toolpad/pull/3077/commits/20755776f3c4e76aaed649ceda54469e735dc31b

The schema definition is much better like this too.